### PR TITLE
[14.0][ADD] shopinvader_product_template_tags

### PR DESCRIPTION
**Pull request by ivantodorovich**
Index Product Template Tags, defined in OCA module `product_template_tags`

Depends on:
- https://github.com/OCA/product-attribute/pull/880

### CODE REVIEW

**Review by simahawk — COMMENTED** [review]
LG, waiting for the dependency to get merged

**Comment by simahawk** [comment]
OCA PR is getting merged

File: test-requirements.txt [hunk 1]
```diff
@@ -7,3 +7,5 @@ vcrpy
 vcrpy-unittest
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+
+odoo14-addon-product_template_tags @ git+https://github.com/OCA/product-attribute@refs/pull/880/head#subdirectory=setup/product_template_tags
```
Comment (simahawk) [inline test-requirements.txt]: you can drop this ;)

**Review by PierrickBrun — APPROVED** [review]
Don't forget to drop the test-requirements

**Comment by ivantodorovich** [comment]
Done! :)

**Pull request merged**